docker-build/fate-upgrade-manager/shutdown-flow.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import time
+from kubernetes import client, config
+
+APP_NAME = "python"
+
+def shutdown_flow(namespace, api, app):
+    if type(app) == client.V1Deployment:
+        update_deployment(namespace, api, app)
+    else:
+        update_sts(namespace, api, app)
+    app = get_flow_app(namespace, api)
+    if app.spec.replicas == 0:
+        print("change the replicas to 0 successfully")
+        # The default grace period is 30 seconds.
+        time.sleep(30)
+        return 0
+    else:
+        print("failed to change the replicas to 0")
+        return -1
```

Comment:
Do we need to wait and check the `status.CurrentReplicas` or `status.ReadyReplicas`?

Reply:
Yeah, that's my first thought. However when I print the whole struct, I find that the status.xxxx is None everywhere.